tests/api/admin/controller/test_admin_search_controller.py

```diff
@@ -101,3 +102,53 @@ def test_search_field_values(self, admin_search_fixture: AdminSearchFixture):
         assert response["languages"] == {"English": 2, "Spanish": 1, "Mandingo": 10}
         assert response["publishers"] == {"Publisher 1": 3, "Publisher 10": 10}
         assert response["distributors"] == {"Gutenberg": 13}
+
+    def test_no_license_counts(self, admin_search_fixture: AdminSearchFixture):
```

Comment:
Nit: a bit confusing method name.

Reply:
:+1: changed this to `test_different_license_types`